base/ltxref.dtx

```diff
@@ -462,10 +463,14 @@
 %    \end{macrocode}
 % \end{plugdecl}
 %    \begin{macrocode}
+\def\ltx@star@counter{*}
```

Comment:
somewhere there should be a `\begin{macro}{\ltx@star@counter}`.

perhaps `\ltx@currentcounter@alias`is a better name? (maybe not)

Reply:
I added macro env,left name as it is